documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl

```diff
@@ -53,7 +53,7 @@ interface Widgets {
   listWidgets is ResourceList<
     Widget,
     {
-      parameters: StandardListQueryParameters & SelectQueryParameter;
+      parameters: StandardListQueryParameters;
```

Comment:
Is the reason of removing this due to SDK generation failure?

Reply:
.Net sdk emitter doesn't support it.